src/cntk-train/src/main/scala/CNTKLearner.scala

```diff
@@ -154,9 +154,10 @@ class CNTKLearner(override val uid: String) extends Estimator[CNTKModel] with CN
     val output = cb.runCommand
     logInfo("CNTK training finished")
     conformedData.unpersist()
+    val modelPath = "file://" + config.getModelPath
```

Comment:
for CNTKLearner it is currently always local, but you're right eventually we want to support both once we have the parquet reader working.